Gaijinizer patch/Map065.txt

```diff
@@ -4,20 +4,21 @@
 
 この地に踏み入る者は、この世の理を踏みしめて進め。
 > CONTEXT: Map065/events/2/pages/0/0/Dialogue
-Here is a company that seals the abyss. 
-Those who step into this land will step on the rule of this
-world.#MTLed
+This is a shrine sealing away the abyss.
+
+Those who walk on this land, tread on the natural order
+of the world.#Ron
 > END STRING
 
 > BEGIN STRING
 『先ず神が天地人を創り、人の理から冥府が生まれた。
 　　　やがて冥府に邪鬼が生まれ、人々を誘惑した
 　　　　この日から、人は生の喜び、死の恐怖を知ったのだ。』
 > CONTEXT: Map065/events/2/pages/0/4/Dialogue
-"First of all, God created the Tenchijin, and from the
-reason of human beings, Sofu was born. Soon an evil spirit
-was born in Sefu and tempted people. From this day, people
-knew the joy of life, the Afraid of death. "#MTLed
+"First, God (神) created the heavens (天), earth, and man...
+And from the minds of men hades (冥) was born... And from
+hades, evil spirits (邪) were born to tempt man. From that
+day man knew the joy of life (生), and terror of death (死)."#Ron
```

Comment:
Hmm.... I wonder if we'd want to call Hades something else here? If we're calling it Hades, we should capitalize all the mentions here.

Reply:
Yeah, it doesn't quite mean hell. It's sort of a "Realm of the Dead".